spec/helpers/content_profiles.ts

```diff
@@ -153,6 +158,49 @@ class ContentProfiles {
                 .first()
                 .element(by.tagName('button')).click();
         };
+
+        /**
+         * Open content profile dropdown.
+        */
+        this.openContentProfileDropdown = () => {
+            element(by.id('content-profile-dropdown')).click();
+        };
+
+        function selectProfile(filtered) {
+            if (filtered.length) {
+                return filtered[1].click();
+            }
+        }
+
+        /**
+         * Returns the list of content profiles.
+        */
+        this.totalProfiles = () => {
+            const dropdownMenu = element(by.id('select-content-profile'));
+
+            return dropdownMenu.all(by.repeater('profile in aggregate.activeProfiles'));
+        };
+
+        /**
+         * Select a content profile from dropdown.
+        */
+        this.selectContentProfile = () => {
+            this.totalProfiles().then(selectProfile);
+        };
+
+        /**
+         * Remove a content profile filter.
+        */
+        this.removeFilter = () => {
+            element(by.className('tag-label__remove')).click();
```

Comment:
I would not use many abstractions in tests. If you want to clear the filters, you do `el('clear-filters').click()` directly from the spec, I don't see a benefit to wrap it in the object. That's the purpose of `data-test-id` tags: even if you change the markup or migrate the component from angular to react, the test tag will still be there. It might be useful to abstract reusable UI components that require multiple steps to do an action like dropdowns/autocomplete/multi-select, but there's not much use abstracting page-specific things that aren't queried from elsewhere.